src/popup/components/AccountSelector.vue

```diff
@@ -41,7 +41,6 @@
       <AddressTruncated
         v-if="!avatarOnly"
         show-explorer-link
-        show-protocol-icon
```

Comment:
What about this prop? Was it also removed by mistake?

Reply:
To be honest I'm not sure. But as the task's scope should not cover this change I'm bringing this line back.